static/app/utils/theme.tsx

```diff
@@ -677,6 +677,31 @@ const commonTheme = {
     lineHeightBody: '1.4',
   },
 
+  /**
+   * Common styles for form inputs & buttons, separated by size.
+   * Should be used to ensure consistent sizing among form elements.
+   */
+  form: {
+    default: {
+      height: '40px',
+      padding: '10px 16px',
```

Comment:
not sure, but shouldn't we use the "spaces" defined in  https://github.com/getsentry/sentry/blob/master/static/app/styles/space.tsx ? 🤷‍♀️

Reply:
many of the sizes here are not available in `space`, e.g. `40px` and `10px`. It's possible that in the future we'll have button padding that strictly adheres to the base-8 sizes from `space`, at which time I'd use `space` here. For now, I'd keep these sizes for the sake of consistency with other components (esp. form inputs).